scripts/onboarding/aks/onboarding-msi-azure-policy/README.md

```diff
@@ -0,0 +1,17 @@
+You can create the policy definition using a command like in windows:
+
+```az policy definition create --name "Preview-AKS-Monitoring-Addon-MSI" --display-name "Preview-AKS-Monitoring-Addon-MSI" --mode Indexed --metadata version=1.0.0 category=Kubernetes --rules .\azure-policy.rules.json --params .\azure-policy.parameters.json```
+
+For linux or wsl environment, change the backward slash to forward slash.
```

Comment:
The command works without slashes as well in both Windows and wsl, can you check that and change the command itself?